pennylane/templates/subroutines/controlled_sequence.py

```diff
@@ -194,3 +211,38 @@ def circuit():
             ops.append(qml.pow(qml.ctrl(base, control=ctrl_wire), z=z, lazy=lazy))
 
         return ops
+
+
+def _cntl_seq_decomposition_resources(base, base_params, num_control_wires) -> dict:
```

Comment:
```suggestion
def _ctrl_seq_decomposition_resources(base, base_params, num_control_wires) -> dict:
```
I think we conventionally use `ctrl` to mean "controlled"